bots/naughty/rhel-7-4

```diff
@@ -0,0 +1 @@
+rhel-7
```

Comment:
At that point we need to split it again; for now it seemed easier to keep the two in sync, but if you prefer I can also copy them right away.